net-dns/https_dns_proxy/https_dns_proxy-0_pre20200925.ebuild

```diff
@@ -20,12 +20,13 @@ HOMEPAGE="https://github.com/aarond10/https_dns_proxy"
 LICENSE="MIT"
 SLOT="0"
 
-DEPEND="dev-libs/libev
+DEPEND="dev-cpp/gtest
```

Comment:
could you work with upstream on making this optional?

Reply:
are you sure this is needed also at runtime, and not just for the test binaries?

Reply:
Yeah, this is just for test binaries, I'll go ahead and fix it now

Reply:
Pushed :)